Snakefile

```diff
@@ -200,25 +200,27 @@ rule clingen:                 # Process the Gene Validity Curations table from C
         opentargets_validator --schema {params.schema} {output.evidenceFile}
         """
 
-rule crispr:                  # Process cancer therapeutic targets using CRISPR–Cas9 screens.
+rule projectScore:                  # Process cancer therapeutic targets using CRISPR–Cas9 screens.
     input:
-        evidenceFile = GS.remote(config['CRISPR']['inputAssociationsTable']),
-        descriptionsFile = GS.remote(config['CRISPR']['inputDescriptionsTable']),
-        cellTypesFile = GS.remote(config['CRISPR']['inputCellTypesTable'])
+        evidenceFile = GS.remote(config['ProjectScore']['geneScores']),
+        cellTypesFile = GS.remote(config['ProjectScore']['cellLinesTable']),
+        uberontoCellLineMapping = HTTP.remote(f"{config['global']['curation_repo']}/{config['Essentiality']['depmap_tissue_mapping']}"),
```

Comment:
Since you've added the logic to deal with the remote file, I'd suggest you move this from `input` (it downloads the data), to `params` (it just creates the environment variable)

Reply:
However this is true for all remote files used in the pipeline.